index.js

```diff
@@ -22,6 +23,15 @@ module.exports = function(schema) {
     }
   });
 
+  Object.keys(schema.virtuals).forEach(function(pathname) {
```

Comment:
`schema.virtuals` might be undefined so you need to check that, that's why the tests fail